server/routes.py

```diff
@@ -8,9 +8,22 @@
 def start_game():
     # Generating a unique game ID
     game_id = str(uuid.uuid4())
+        
+    # Create the llm to use for this game
+    # todo:: get the user's provided API key ;)
```

Comment:
I'm down with us handling this another time, will add it to the Issues tab.

Reply:
Smart